src/decode.c

```diff
@@ -403,6 +403,7 @@ void DecodeRegisterPerfCounters(DecodeThreadVars *dtv, ThreadVars *tv)
     dtv->counter_avg_pkt_size = StatsRegisterAvgCounter("decoder.avg_pkt_size", tv);
     dtv->counter_max_pkt_size = StatsRegisterMaxCounter("decoder.max_pkt_size", tv);
     dtv->counter_erspan = StatsRegisterMaxCounter("decoder.erspan", tv);
+    dtv->counter_flow_memcap = StatsRegisterCounter("decoder.flow_failed", tv);
```

Comment:
The flow memcap counter is called "decoder.flow_failed". Thats not very consistent. I think it's fine to just call it 'flow.memcap' so that it matches the http.memcap format.